requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy>=1.15
+PySC2>=2.0
```

Comment:
Maybe use the git version, the pip one is very outdated like so:
```
git+https://github.com/deepmind/pysc2.git@a9f093493c4c77adb385602790a480e7f238b97d
```

Reply:
Good point, I will replace it

Reply:
I guess we could add PyTorch as a dependency, unless somebody has strong preference again (voted 4:4 yesterday)